x-pack/plugins/monitoring/public/components/apm/instance/instance.js

```diff
@@ -6,66 +6,28 @@
  */
 
 import React from 'react';
-import { MonitoringTimeseriesContainer } from '../../chart';
-import {
-  EuiFlexItem,
-  EuiPanel,
-  EuiSpacer,
-  EuiPage,
-  EuiPageBody,
-  EuiFlexGroup,
-  EuiPageContent,
-  EuiScreenReaderOnly,
-} from '@elastic/eui';
-import { Status } from './status';
-import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+import { ApmMetrics } from '../apm_metrics';
 
-export function ApmServerInstance({ summary, metrics, ...props }) {
+const title = i18n.translate('xpack.monitoring.apm.instance.panels.title', {
+  defaultMessage: 'APM Server - Custom Metrics',
```

Comment:
"APM Server - Metrics" rather than "APM Server - Custom Metrics" because these metrics are not cusom in the sense of defined by the user but they are the metrics we recommend to monitor APM Server.